simplq/src/components/pages/Status/QueueStatus.jsx

```diff
@@ -13,17 +13,16 @@ import QueueDetails from './QueueDetails';
 const TIMEOUT = 10000;
 let timeoutId;
 
-function QueueStatus() {
+function QueueStatus(props) {
+  const tokenId = props.match.params.tokenId;
+  const [tokenStatusResponse, setTokenStatusResponse] = useState();
   const dispatch = useDispatch();
-  const queueName = useSelector((state) => state.appReducer.queueName);
-  const [tokenStatus, setTokenStatus] = useState();
-  const tokenId = useSelector((state) => state.appReducer.tokenId);
-  const aheadCount = useSelector((state) => state.appReducer.aheadCount);
+  dispatch(setJoinerStep(2));
```

Comment:
I'm wondering if instead of using magic numbers, we should define meaningful constants.

Reply:
Seems like too many imports would be required in a lot of places. If that is the case, what if we just a description of what each number means in the setJoinerStep and setCreatorStep functions atleast as a temporary measure.

Reply:
Imports are fine. We should use constants, I feel.